python/idsse_common/idsse/common/rabbitmq_utils.py

```diff
@@ -98,6 +98,12 @@ class PublishMessageParams(NamedTuple):
     route_key: str | None = None
 
 
+class RpcResponse(NamedTuple):
```

Comment:
Do we need this class in addition to PublishMessageParams?

Reply:
Technically no, but `PublishMessageParams` is used for outbound RMQ messages and `RpcResponse` was intended for inbound responses. I just don't know what to name the class if it's used for both.

Reply:
Maybe "RabbitMqMessage"?

Reply:
That works, or RmqMessage if you want shorter.